src/dev/jest/babel_transform.js

```diff
@@ -32,9 +32,4 @@ module.exports = babelJest.createTransformer({
       },
     ],
   ],
-  plugins: [
-    // enables jest to parse and execute dynamic import() calls
```

Comment:
Aren't these still required for monaco support?

Reply:
I haven't seen any test failures related to monaco with these changes. It's possible that monaco is just not hit by Jest